spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe Api::V1::RentsController do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    let!(:rents) { create_list(:rent, 3, user: user) }
+
+    before { get :index, params: { user_id: user.id } }
+
+    it { expect(response).to have_http_status(:ok) }
+
+    it 'responds with the user\'s rents in json format' do
+      expected = ActiveModel::Serializer::CollectionSerializer.new(
+        rents, each_serializer: RentSerializer, root: false
+      ).to_json
+
+      expect(response_body.to_json) =~ JSON.parse(expected)
+    end
+  end
+
+  describe 'POST #create' do
+    let!(:book) { create(:book) }
+    let!(:rent_attrs) { attributes_for(:rent) }
+
+    before do
+      post :create, params: {
+        user_id: user.id,
+        rent: rent_attrs.merge(book_id: book.id)
+      }
+    end
+
+    it { expect(response).to have_http_status(:created) }
+
+    it 'responds with the serialized new rent' do
+      expect(response_body['start_date'].to_date).to eq rent_attrs[:start_date]
+      expect(response_body['book']['id']).to eq book.id
```

Comment:
Careful with this. 
> Each test should make only one assertion.
[¨Single expectation test¨](http://www.betterspecs.org/)

Reply:
@lmasello I did read that, but I couldn't think of a better way to write this test with a single expectation. Should I pick one of the three expectations and test only for that?

Reply:
Or you can test them separately, like:
`it 'responds with the right book'`
And similar with the user.

Reply:
Good. I'll do that.